changelog/15369.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+mfa/okta: disable client side rate limiting causing delays in push notifications
```

Comment:
I'd think that `auth/okta` refers to the Okta auth method specifically.

Reply:
Agreed. Not sure how to categorize login mfa.

Reply:
I am good either way. I just mentioned that because we used `auth` as the category for login MFA before, and then message was more specifically talked about what has changed in Login MFA.